Makefile

```diff
@@ -0,0 +1,23 @@
+SHELL=/bin/bash -o pipefail
+.PHONY: local remote deploy
+
+remote: index.bs
+	@ (HTTP_STATUS=$$(curl https://api.csswg.org/bikeshed/ \
+	                       --output index.html \
+	                       --write-out "%{http_code}" \
+	                       --header "Accept: text/plain, text/html" \
+	                       -F die-on=warning \
+	                       -F md-Text-Macro="COMMIT-SHA LOCAL COPY" \
+	                       -F file=@index.bs) && \
+	[[ "$$HTTP_STATUS" -eq "200" ]]) || ( \
+		echo ""; cat index.html; echo ""; \
+		rm -f index.html; \
+		exit 22 \
+	);
+
+local: index.bs
+	bikeshed spec index.bs index.html --md-Text-Macro="COMMIT-SHA LOCAL COPY"
+
+deploy: index.bs
+	curl --remote-name --fail https://resources.whatwg.org/build/deploy.sh
```

Comment:
Not a blocker, but just FYI: I am a bit uneasy with having a Makefile rule that downloads code from the internet and runs it. Can this rule maybe just spit out instructions to the terminal that can be copy-pasted?

```
deploy: index.bs
        echo \
To deploy changes, run the following: \
\
curl --remote-name --fail https://resources.whatwg.org/build/deploy.sh && bash ./deploy.sh
```

... just to make sure the user is aware?

Reply:
This was "inspired" by the [DOM spec makefile](https://github.com/whatwg/dom/blob/main/Makefile) and thought it would be nice to have it for Intersection Observer spec to allow building without having to install bikeshed locally.
I'll remove the `deploy` part.